ansible/roles/connectbox-pi/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set keyboard to the US layout
+  lineinfile:
+    path: /etc/default/keyboard
```

Comment:
`path` was introduced in Ansible 2.3. We’re using 2.1, so you’ll need to use the old parameter `dest`. It has the same effect.